plasma_framework/test/src/exits/payment/controllers/PaymentProcessInFlightExit.test.js

```diff
@@ -464,6 +465,14 @@ contract('PaymentProcessInFlightExit', ([_, ifeBondOwner, inputOwner1, inputOwne
                 expect(postBalance).to.be.bignumber.equal(expectedBalance);
             });
 
+            it('should return piggyback bond to the output owner', async () => {
```

Comment:
is there testing that a challenged invalid piggyback doesn't return the bond? Also are both canonicity-cases covered in here?